frontend/src/ui-kit/components/Playground/Playground.test.tsx

```
@@ -0,0 +1,229 @@
+      defaultValue: 'Click me',
+    },
+  ];
+
+  // Mock scope with a simple Button component
+  const mockScope = {
+    Button: ({ variant, disabled, children }: {
+      variant?: string;
+      disabled?: boolean;
+      children: React.ReactNode;
+    }) => (
+      <button data-variant={variant} disabled={disabled}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)